src/index.html

```diff
@@ -2,11 +2,331 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/reset.css"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="header-container">
+      <header class="header">
+        <a
+          href="index.html"
+          class="logo"
+        >
+          <img
+            class="logo_img"
+            src="./images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list list">
+            <li class="list__item nav-item">
+              <a
+                class="nav-item__link is-active"
+                href="#Apple"
+              >Apple
+              </a>
+            </li>
+            <li class="list__item nav-item">
+              <a
+                class="nav-item__link"
+                href="#Samsung"
+              >Samsung
+              </a>
+            </li>
+            <li class="list__item nav-item">
+              <a
+                class="nav-item__link"
+                href="#Smartphones"
+              >Smartphones
+              </a>
+            </li>
+            <li class="list__item nav-item">
+              <a
+                class="nav-item__link"
+                href="#computers"
+                data-qa="nav-hover"
+              >Laptops & Computers
+              </a>
+            </li>
+            <li class="list__item nav-item">
+              <a
+                class="nav-item__link"
+                href="#Gadgets"
+              >Gadgets
+              </a>
+            </li>
+            <li class="list__item nav-item">
+              <a
+                class="nav-item__link"
+                href="#Tablets"
+              >Tablets
+              </a>
+            </li>
+            <li class="list__item nav-item">
+              <a
+                class="nav-item__link"
+                href="#Photo"
+              >Photo
+              </a>
+            </li>
+            <li class="list__item nav-item">
+              <a
+                class="nav-item__link"
+                href="#Video"
+              >Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+    </div>
+    <main class="contant">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__img">
+        </div>
+        <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock
+          (MNED2UA/A)
+        </div>
+        <div class="card__product-code">Product code: 195434
+        </div>
+        <div class="card__reviews reviews">
```

Comment:
```suggestion
        <div class="card__reviews">
```
It could be just an element of card block here

Reply:
this element has children, i had to make it a block for lint

Reply:
or i dont understand you